RevenueCat/Plugins/iOS/PurchasesUnityHelper.m

```diff
@@ -226,7 +214,7 @@ - (char *)getAppUserID {
 }
 
 - (BOOL)isAnonymous {
-    return @([RCCommonFunctionality isAnonymous]);
+    return RCCommonFunctionality.isAnonymous;
```

Comment:
this was a bug - doing the `@([` on a boolean always evaluates to true

Reply:
😮‍💨